rfcs.md

```diff
@@ -33,3 +33,55 @@ Some changes that would not need an RFC:
 The goal of our RFC process is to reach a consensus among the cauldron team on proposed changes, and to ensure that new features are congruent with the rest of Cauldron. The Cauldron team will have a recurring office hours meeting on the 1st Wednesday of each month, giving the team the opportunity to discuss any new RFCs that have been introduced and make comments.
 
 Once a consensus has been reached for the proposed RFC, the `rfc` label will be removed from the issue and the proposed changes can then be implemented.
+
+## What is a Breaking Change?
```

Comment:
@dequejosie Thanks for putting this together. However, this document is just meant to document our RFC process--not actually contain the RFC's. To submit an RFC, you should follow the steps listed out [above](https://github.com/dequelabs/cauldron/blob/develop/rfcs.md#how-to-submit-an-rfc):

1. Create a new issue
1. Add the `rfc` label
1. Be sure to include descriptive details, this includes:
   - A descriptive title
   - Implementation details
   - Screenshots (if necessary for a new component)